messages/create.json

```diff
@@ -0,0 +1,35 @@
+{
+  "description": "creates an Experience Cloud site using a template\nSee 'Which Experience Cloud Template Should I Use?' in Salesforce Help for more information about the different template types available for Experience Cloud.\n\nWhen creating a site with the Build Your Own (LWR) template, you must also specify the AuthenticationType value using the format templateParams.AuthenticationType=value, where value is AUTHENTICATED, UNAUTHENTICATED, or AUTHENTICATED_WITH_PUBLIC_ACCESS. Name and values are case-sensitive. See 'ExperienceBundle' in the Metadata API Developer Guide for more information.\n\nWhen you execute this command, it creates the site in preview status, which means that it isn’t yet live. After you finish building your site, you can make it live.\n\nIf you have an Experience Builder site, publish the site using the sfdx force:community:publish command to make it live.\n\nIf you have a Salesforce Tabs + Visualforce site, activate the site to make it live by updating the status field of the Network type in the Metadata API. Alternatively, in Experience Workspaces, go to Administration | Settings, and click Activate.\n\nFor Experience Builder sites, activating the site just sends out a welcome email to site members.",
+  "longDescription": "Creates an Experience Cloud site using a template.",
```

Comment:
(I expound on this below) -- I htink you can remove this property, for neatness, because it's ignored.